ext/hash/config.m4

```diff
@@ -73,6 +106,8 @@ PHP_INSTALL_HEADERS([ext/hash], m4_normalize([
   php_hash_ripemd.h
   php_hash_sha.h
   php_hash_sha3.h
+  php_hash_blake3.h
+  blake3/upstream_blake3/c/blake3.h
```

Comment:
Why is there that extra upstream_blake3 directory. Cannot this go directly to blake3?

Reply:
the directory separate code written for the php-src integration from the substantial amount of code cloned directly from https://github.com/BLAKE3-team/BLAKE3/

Reply:
I don't really understand why it needs extra subdirectory. The blake3 directly contains only upstream_blake3 directory which really doesn't make any sense to me. And you also have c directory which I can see is from upstream. So it should really be just blake3/c .